src/plugins/mam/Archive.ts

```diff
@@ -13,7 +13,7 @@ import MultiUserContact from '@src/MultiUserContact';
 
 export default class Archive {
    private archiveJid: IJID;
-   private messageCache: JQuery<HTMLElement>[] = [];
+   private messageCache: any[] = [];
```

Comment:
That's never a good idea. Seams like a source for massive issues in the future.

Reply:
i created a simple class for that now